docs/jumpstart/week-10.md

```diff
@@ -0,0 +1,221 @@
+
+### Why do we use Heroku
+The main reason we're going to use Heroku is because it is easier to use than traditional cloud deployment platforms, as previously mentioned, and it allows us to deploy our application for free. When you deploy your application you need some computing resources. These computing resources cost money, therefore just to run a very simple server with minimum hardware capabilities can cost around 5-10 dollars a month. Heroku allows us to use these minimum computing resources for free. Of course if you want to upgrade your computing resources to use more RAM or CPU you can pay for that option.  
+
+### Install Heroku
+The way our application is structured, our frontend and backend are separated. They are not dependent on one another and are developed in different environments and servers. Therefore, we have to deploy our frontend and backend separately. Thankfully, Heroku provides great documentation as to how to deploy our React frontend application and our Java backend. However, before you deploy any application on Heroku we need to install Heroku on our machine. First, we need to create an account on Heroku. The link to sign up is https://signup.heroku.com/. Once you have signed up for a Heroku account, you have to install it on your machine. Because there are different installation methods depending on the machine that you are using, the download instructions for every operating system can be found here https://devcenter.heroku.com/articles/heroku-cli. What you are installing is Heroku's command line application which will allow you to open a terminal and run heroku commands on the terminal.
+
+### Deploy Frontend
+To deploy our frontend application, we need to navigate to the directory that contains our Frontend application. Once we have navigated to our directory we have to initialize a git repository if a git repository has not been initialized yet. To initialize a git repository we run the command:
```

Comment:
Can you fix this line to either have frontend as capitalized or not (it looks like you keep it lowercase everywhere else, so maybe just do that)